sphinx_gherkindoc/cli.py

```diff
@@ -146,7 +147,7 @@ def process_args(
 
     if step_glossary_name:
         glossary_filename = output_path / f"{step_glossary_name}.rst"
-        glossary = make_steps_glossary(doc_project)
+        glossary = make_steps_glossary(doc_project, group_step_glossary)
```

Comment:
First quick scan of this pr, this new parameter is a keyword parameter, so please pass it as such so that we maintain the flexibility to add other optional keyword parameters in the future and don't have to worry about the positional->keyword mappings being correct, thanks!